### ISSUE TITLE
ExportToNewSyntax error when running New-CSideDevContainer

### ISSUE THREAD

**Issue by ewaldventer** [Microsoft/nav-docker#99]
Hi,
Dev environment and Windows Client runsperfect, but encounter the below error during setup.

```
WARNING: UnhandledErrorMessage
A parameter cannot be found that matches parameter name 'ExportToNewSyntax'.
At C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.1.1.9\NavContainerHelper.psm1:1087 char:5
+     Invoke-Command -Session $session -ScriptBlock { Param($filter, $o ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Export-NAVApplicationObject], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Export-NAVApplicationObject
    + PSComputerName        : c3a63fbd9aea4f02f68480576bbab3602aa24627f6a2810c5707c9783727b235

```

**Comment by freddydk** [comment]
This happens if you are running NAV 2017 or earlier. Will be fixed with the next update on the PowerShell Gallery.

**Comment by freddydk** [comment]
Microsoft/navcontainerhelper#16

**Comment by freddydk** [comment]
This is fixed in navcontainerhelper 0.1.2.0
Use
update-module navcontainerhelper -force
to update
